client/app/types/course/assessment/question/voice-responses.ts

```diff
@@ -0,0 +1,27 @@
+import { AvailableSkills, QuestionFormData } from '../questions';
+
+type OptionalIfNew<T extends 'new' | 'edit'> = T extends 'new'
```

Comment:
This seems to be repeated in `multiple-responses.ts`. Can you abstract this to `questions.ts`?